src/index.js

```diff
@@ -438,6 +438,39 @@ class AnalyticsCoreAPI {
     })
   }
 
+  /** Retrieves usage and access logs for the search criteria provided.
+  * This API returns the usage and access logs for a given date range within a 3 month period. This API authenticates with an IMS user token.
+  *
+  * @param startDate {string} Start date for the maximum of a 3 month period.
```

Comment:
startDate and endDate need to be in ISO-8601 format don't they? We should specify this in the JSDoc